apps/nova/src/lib/ratings.ts

```diff
@@ -0,0 +1,50 @@
+import { IRating } from "@litespace/types";
+import { TutorRatingCardGroupProps } from "@litespace/luna/TutorFeedback";
+
+export const organizeRatings = (
```

Comment:
The implementation doesn't match [this](https://github.com/litespace-org/litespace/pull/224#discussion_r1893506218).